packages/labs/ssr-dom-shim/src/index.ts

```diff
@@ -4,12 +4,24 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 import {ElementInternalsShim} from './lib/element-internals.js';
+import {CSSStyleSheetShim} from './lib/css.js';
 
 export {
   ariaMixinAttributes,
   ElementInternals,
   HYDRATE_INTERNALS_ATTR_PREFIX,
 } from './lib/element-internals.js';
+export {
+  CSSRule,
+  CSSRuleList,
+  CSSStyleSheet,
+  MediaList,
+  StyleSheet,
+} from './lib/css.js';
+
+// We want to provide CSSStyleSheet in the global scope, as
+// `new CSSStyleSheet()` can be called anywhere in consumer code.
+globalThis.CSSStyleSheet ??= CSSStyleSheetShim;
```

Comment:
I'm not opposed to this, but is the philosophy behind this documented somewhere? My first instinct would be to make it "just work", as moving it into `getWindow()` requires consumers to first find and use this functionality, in contrast to e.g. customElements being globally available.

Reply:
The globals are defined `getWindow()` so that they can be installed into different context than the one that imports the shim. Unless and until we remove support for VM modules we need to keep that ability. `install-global-dom-shim.js` exists for when you want to automatically install the globals into the importing context.

Reply:
btw, the `customElements` global is installed in `reactive-element.ts` (in addition to `installWindowOnGlobal()`) in the Node build: https://github.com/lit/lit/blob/main/packages/reactive-element/src/reactive-element.ts#L60

We _could_ add that there, but the reason why `customElements` is there is because it's so common and we're just trying to make the basic element load in Node without errors. `new CSSStyleSheet()` is much less comment, and importing CSS less so since it's only supported in Chrome.

Reply:
Yes, I noticed that. But yeah, that is reasonable.